### PR TITLE
Add a color theme switcher parameter, add css themes

### PR DESCRIPTION
**Pull request by drmenzelit**
Pull Request for Issue #51 .

### Summary of Changes
Added a new parameter: Color Theme
Created new scss files for the global definitions of colors as css variables for each theme
Load css depending on the selected theme

### Testing Instructions

Install PR and run npm ci or complie scss

Change the theme in the template configuration and reload the page. You should see that links and buttons change the colors depending on the selected theme.

**Autumm**
Screenshot_2020-09-04 Home-autumm

**Spring**
Screenshot_2020-09-04 Home-spring

**Summer**
Screenshot_2020-09-04 Home-summer

**Winter**

Screenshot_2020-09-04 Home-winter

### CODE REVIEW

File: language/en-GB/tpl_cassiopeia.ini [hunk 1]
```diff
@@ -5,6 +5,11 @@
 
 TPL_CASSIOPEIA_BACKTOTOP="Back to Top"
 TPL_CASSIOPEIA_BACKTOTOP_LABEL="Back-to-top Link"
+TPL_CASSIOPEIA_COLOR_NAME_LABEL="Color Theme"
```
Comment (brianteeman) [inline language/en-GB/tpl_cassiopeia.ini]: it is colour

**Comment by brianteeman** [comment]
Please check the colours for contrast - they are all failing for accessibility

You can use this tool to check https://webaim.org/resources/contrastchecker/

image

**Comment by chmst** [comment]
The colours are not in scope of this PR, here it is only the parameter itself.

**Comment by bembelimen** [comment]
See: https://github.com/joomla/cassiopeia/pull/57#pullrequestreview-482553488

File: language/en-GB/tpl_cassiopeia.ini (hunk 1 above)
Comment (drmenzelit) [reply language/en-GB/tpl_cassiopeia.ini]: I forgot we use British English

**Comment by richard67** [comment]
@drmenzelit Could we maybe rename the scss files in the "global" folder e.g. from "spring.scss" to something like "color_scheme_spring.scss" or "color_spring.scss"? The reason is that we could add more scss files for other purpose, too, e.g. selection of a font combination (I think about modifying the "google fonts" stuff). If we can agree a naming scheme and you are too busy I can make a PR for you to the branch of this PR.

**Comment by richard67** [comment]
@drmenzelit Another thing: There are no css files build from the scss files in the "global" folder.

And you use "autumm", which is an ugly typo, it should be "autumn".

I have made a PR for you: #59 .

Just check that and let me know if it's ok, then I will merge it, or you can merge it on GitHub usint the button at the bottom.

**Comment by richard67** [comment]
Issues mentioned in my comments above are solved.

**Review by richard67 — APPROVED** [review]

**Review by hans2103 — APPROVED** [review]

**Pull request merged**